examples/pytorch/graphsage/node_classification.py

```diff
@@ -3,130 +3,131 @@
 import torch.nn.functional as F
```

Comment:
Is this script simply sampling-based version of `train_full.py`? If so, the naming can also be confusing.

Reply:
Yes, it is confusing and quite annoying... we definitely need to fix.
Alternative 1: train_full.py  train_sampling.py, but in this way, link_pred.py is an outlier which is named after type of task..
Alternative 2: node_classification_full.py, node_classification.py link_pred.py.   these sound less confusing ; but quite annoying to adopt such convention to other examples (i.e., always name after type of task) ......

I wanna defer this until I finished updating link_pred.py unless you have better proposal(s) :)

Reply:
Sure. We don't necessarily need to have the same file names across all examples. As you said, we can decide later.